borneodp.rb

```diff
@@ -5,11 +5,11 @@
 class Borneodp < Formula
   desc ""
   homepage "https://github.com/borneodata/homebrew-borneodp"
-  version "3.7.0"
+  version "3.8.0"
 
   on_macos do
     url "https://github.com/borneodata/homebrew-borneodp/releases/download/v3.7.0/deploy-dataplane_Darwin_all.tar.gz"
```

Comment:
The download URL for all binaries needs to change as well, right?
```suggestion
    url "https://github.com/borneodata/homebrew-borneodp/releases/download/v3.8.0/deploy-dataplane_Darwin_all.tar.gz"
```